product_docs/docs/pgd/5/rel_notes/pgd_5.1.0_rel_notes.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "Release notes for EDB Postgres Distributed version 5.1.0"
+navTitle: "Version 5.1.0"
+---
+
+EDB Postgres Distributed version 5.1.0 is a patch version of EDB Postgres Distributed.
+This version addresses security vulnerabilities in dependencies for PGD-Proxy and PGD-CLI.
+
+| Component | Version | Type            | Description                                          |
```

Comment:
Missing from this list are:
- consistent use of tcp keepalives across all BDR connections ( 77e3df56010c2301c573ef8c63f9c043067f1b4c )
- timeout for inactive Raft connections, same reasoning and benefits as wal_sender_timeout/wal_receiver_timeout  ( 234e9f718774531be85a9f63a49c7e867bfdb8d5 )
- making backends that receive Raft messages easily identifiable ( 8d29a110ba6d35bc284afea38352cf1e5ea2206b )